pkg/agent/baker.go

```diff
@@ -1005,6 +1005,9 @@ func getContainerServiceFuncMap(config *datamodel.NodeBootstrappingConfiguration
 		"GetBootstrapProfileContainerRegistryServer": func() string {
 			return config.ContainerService.Properties.SecurityProfile.GetPrivateEgressContainerRegistryServer()
 		},
+		"GetMCRRepositoryBase": func() string {
+			return config.CloudSpecConfig.KubernetesSpecConfig.MCRKubernetesImageBase
```

Comment:
consider also adding a case to baker_test.go to make sure we're always getting the correct value - non-blocking

Reply:
Agree with this :)

Reply:
sure, will do in another PR.